src/django/api/serializers.py

```diff
@@ -634,9 +657,9 @@ def get_other_locations(self, facility):
                 'contributor_id':
                 l.facility_list_item.source.contributor.admin.id
```

Comment:
While testing I discovered that we have an additional requirement that was not captured in the original issue.

The name of the contributor is being correctly anonymized, but we are still creating a clickable link to the profile page where the full contributor details are visible. I think this is easily taken care of by adding a `user_can_see_detail` check here and making a complimentart change to the client code that gracefully handles a null `contributor_id`

NOTE this screencast has a map-related soft crash that is not related to the anonymization issue.

![2021-07-06 14 43 45](https://user-images.githubusercontent.com/17363/124670685-33632500-de69-11eb-8d9a-64535e1c8f87.gif)